pcap/pcap.go

```diff
@@ -64,11 +66,20 @@ func (p *pcapImpl) capturePackets(done <-chan struct{}, interfaceName, bpfFilter
 				if ok {
 					wrappedChan <- pkt
 
-					if count == 0 {
+					if firstPacket {
+						firstPacket = false
 						ttfp := time.Since(startTime)
 						printer.Debugf("Time to first packet on %s: %s\n", interfaceName, ttfp)
 					}
-					count += 1
+
+					now := time.Now()
```

Comment:
I think this measurement needs to go above the call to `wrappedChan <- pkt` above.  That might block (why did we decide to make the channel only 10 packets?) and that possibility seems to mess with our measurement.